public/api/conf/1.0/schemas/subSchemas/accountingPeriod.json

```diff
@@ -8,11 +8,11 @@
   ],
   "properties": {
     "startDate": {
-      "description": "The start date of the period of account. It cannot be before 2016-10-01",
+      "description": "The start date of the period of account. It cannot be before 2016-10-01. For example: 2020-04-06",
       "$ref": "../definitions/definitions.json#/definitions/dateType"
     },
     "endDate": {
-      "description": "The end date of the period of account. It must be within 18 months after the start date",
+      "description": "The end date of the period of account. It must be within 18 months after the start date. For example: 2020-04-06",
```

Comment:
Not sure about this hardcoded date? ie, what happens if someone triggers this error when start date was 2020-05-10??